### DISCUSSION TITLE
Could you please explain me what’s this issue?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35588] [Projects and Issues]
![image](https://user-images.githubusercontent.com/24357017/194753223-4252b9ec-8f60-491e-b2fc-6b961d83933c.png)

**Comment by dennewbie** [comment]
If you're referring to the red one as the error, it seems like it's related to this [issue](https://github.com/orgs/community/discussions/35590). However, that's still without a solution, as well. Furthermore, I found a similar [discussion](https://github.com/community/community/discussions/29203) here on GitHub.
